tarantool/response.py

```diff
@@ -248,3 +251,34 @@ def __str__(self):
         return ''.join(output)
 
     __repr__ = __str__
+
+
+class ResponseExecute(Response):
+    @property
+    def lastrowid(self):
+        if self.body is None:
+            raise InterfaceError("Trying to access data, when there's no data")
+        info = self.body.get(IPROTO_SQL_INFO)
+
+        if info is None:
+            return None
+
+        lastrowids = info.get(IPROTO_SQL_INFO_AUTOINCREMENT_IDS)
+
+        return lastrowids[-1] if lastrowids else None
+
+    @property
+    def rowcount(self):
```

Comment:
How about name it `affected_row_count` to make things clear? There is `rowcount` property in Response and I'm a bit fear about ambiguity.

Reply:
I did the same for rowcount as for lastrowid to correspond Tarantool naming for this layer.